### DISCUSSION TITLE
Tübitak 2204a Proje Örnekleri

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/sonsuzus#456] [Fikirler]
Bu başlık altında her proje öğrencisi bir projeyi bulup ekleyecek. Eklediği proje daha önce eklenen projelerden olmayacak ve ayrıca Yazılım bölümünden olacak.

Projeleri bulacağınız link: https://2204a.tubitak.gov.tr/

**Comment by sonsuzus** [comment]
### ULUSAL PARMAK İZİ TOPLU TAŞIMA SİSTEMİ
Bu proje kapsamında toplu taşımada ödeme yapmak için Radyo Frekansı ile Tanımlama (RFID) teknolojisini kullanan akıllı kartların yerine parmak izini kullanan ulusal parmak izi toplu taşıma sistemi (UPARTUS) geliştirildi. Öncelikle literatür taraması ile parmak izinin ulaşımda kullanım alanları araştırıldı. Daha sonra Türkiye'deki 81 ilin merkez belediyelerinin kullandığı toplu taşıma ödeme sistemleri araştırıldı. Çoğu ilde RFID tabanlı akıllı kartlar kullanılmaktadır. Bazı illerin aynı firma ile çalıştığı aynı kartın farklı şehirlerde farklı isimler altında kullanıldığı tespit edildi. Bir kartın maliyeti yaklaşık 0,7 TL'dir. Ülkemizde 0-6 yaş arası çocuklar ücretsiz toplu taşımayı kullanırlar. Türkiye nüfusu ise 2017 TÜİK verilerine göre 80 milyon 810 bin 525'tir. Bu durumda 75 milyon 677 bin 113 kişinin toplu ulaşım kartı kullandığı varsayılırsa kart maliyeti yaklaşık 52 milyon 973 bin 979,1 TL'dir. Tasarladığımız UPARTUS sistemi ile kart için harcanan bedel sıfırlanacaktır ve Türkiye büyük tasarruf sağlayacaktır. Çünkü sistemde ödeme için parmak izi kullanılmaktadır. Ayrıca parmak izi kişiye özeldir. İnsanların harici bir kart vb. materyal taşımasına gerek yoktur. Sadece parmak izi okuyucu sistemin tasarlanması yeterlidir. UPARTUS'un arduino mikrodenetleyici kartı, parmak izi sensörü, LCD ekran, potansiyometre, bağlantı kabloları, kızıl ötesi alıcı ve kumanda ile prototipi yapıldı. Prototip üzerinde hem kayıt işlemi hem de okutma işlemi yapılabilmektedir. Kayıt işlemi örnek kişisel veriler üzerinden çalışmaktadır. Gerçek sistemdeki hedefimiz kişisel verilerin MERNİS sisteminden alınması ödemenin ise HGS veya OGS'de olduğu gibi kişinin banka hesabından düşmesi yönündedir. Kişi parmak izini bir kere sisteme kaydettirdikten sonra ülkenin herhangi bir ilinde toplu taşımayı kullanmak istediğinde parmağını UPARTUS okuyucusuna dokunması yeterlidir.

**Comment by Ersan0102** [comment]
**_KAĞITTAN UÇAK SİMÜLASYON PROJESİ_**

### Kâğıttan Uçak Simülasyonu projesi okullarda öğretilmeyen ve günlük hayatta bizimle iç içe olan aerodinamik kavramlarını öğreterek Havacılık ve Uzay sektörüne olan ilgiyi arttırmayı amaçlamaktadır. Bu hususta, erişimi basit ve test imkânı fazla olan kâğıttan uçağa odaklanılmıştır. Ürün olarak içerisinde aerodinamik kavramlar için interaktif bir öğrenme platformu ve 3 boyutlu kâğıttan uçak simülasyonu olan bir bilgisayar programı geliştirilmiştir. Bu program geliştirilirken 100'ün üzerinde uçak modelinde 1000'den fazla uçuş denemesi yapılmış ve veriler yapay zekâ koduna aktarılmıştır. Simülasyon içerisinde tasarlanan kâğıttan uçağın uçuş mesafesi ve süresi gibi bilgiler %82'lik doğru oranında gerçekle örtüşmektedir. Program oluşturulduktan sonra 14-18 yaşları arasında 35 kişilik test grubu üzerinde incelemeler yapılmıştır. Bu incelemeler sonucunda simülasyonu kullanmadan önce ve kullandıktan sonraki kâğıt uçakların uçuş mesafeleri kıyaslandığında %209 artış gözlemlenmiştir. Yapılan anketler sonucunda öğrencilerin aerodinamiğe ilgisinin halihazırda bulunduğu ve bilgilerini yanlış bir tavırla simülasyondan önce yeterli buldukları saptanmıştır. Simülasyondan sonra ise öğrencilerin cevapları değişmiş ve Havacılık ve Uzay endüstrisinde çalışmaya daha meyilli oldukları kaydedilmiştir.

**Comment by FurkanYildirim1350** [comment]
****KİMYA DERSİNDE ARTIRILMIŞ GERÇEKLİK UYGULAMASI VE MOLEKÜL MODELLERİ****
**Bu projede, teknolojik yaklaşımları kullanılarak kimya dersine olan ilgi nasıl arttırılabilir sorusuna cevap aranmıştır. Özelikle organik kimya ders kitaplarında bulunan molekül modellerini arttırılmış gerçeklik sistemiyle görsel bir şölen halinde eğlenceli anlaşılır bir öğrenme motivasyonun oluşturulması hedeflenerek, hayal dünyasını geliştiren, öğreten ve öğrencilerin ders verimini arttıran görsel zekayı hedefleyerek öğrencileri bir üst seviyeye taşıyacağına inandığımız bu programı geliştirdik. Gerçek ortama yansıtmak için kullanılan malzemeleri (resim ve üç boyutlu model) tasarlanıp modellenmesi için gerekli olan yazılım ve donanımlar hazırlandı. Visual Studio ile kodların hazırlanması gerçekleşti. Bu kodlar sayesinde oluşturduğumuz molekül modellerinin hareketi (sağ, sol, yukarı, aşağı) ve yakınlaştırma (büyültüp, küçültme) özellikleri sağlandı. Öğrencilerin görsel zekalarını ve hayal gücü yeteneklerini daha fazla geliştirmek için Cinema 4D ile renklendirme yaptık. Daha sonra topladığımız bütün verileri Unity 3D üzerinden asıl amacımız olan arttırılmış gerçeklik uygulamasına dönüştürdük. Ve son olarak Android Studio ile sağlanan SDKlar ile birlikte androidi destekleyen mobil cihazlara uygun hale getirdik.**

**Comment by Sadik1Dev** [comment]
KANTİNİMDE SIRA YOK - SÜPER KANTİN
Okullarda eğitim-öğretim süreçlerinin verimli bir şekilde tamamlanması için önemli bir unsurdur teneffüs araları. Bu aralar öğrencilerin dinlendikleri, arkadaşları ile sosyalleştikleri ve oyun oynadıkları önemli zaman dilimleridir (Genç, 2003, 1-2). Bu zaman diliminde öğrenciler bazı temel ihtiyaçlarını kantin hizmetlerinden yararlanarak karşılamaktadır. Kısıtlı sürede yüzlerce öğrencinin aynı kantin hizmetlerine başvurması, sıra ve kuyrukların oluşmasına neden olmaktadır. Kantin hizmetlerinden yararlanmak için kuyrukta bekleyen öğrenciler, sınırlı süre içinde alışverişlerini ya yetiştirememekte ya da geç gerçekleştirip elindeki yiyecek ve içecekle derse girmek zorunda kalmaktadırlar. Bu süreç eğitim-öğretimi olumsuz etkilemektedir. Diğer taraftan kantin hizmetlerini sunan personeller, kısıtlı teneffüs süresinde hızlı hareket etmek zorunda olduklarından dolayı hijyen ve sağlıklı kurallarına uymakta zorlanmaktadırlar. Öğrencilerin kısıtlı teneffüs sürelerinde kantin alışverişlerini sıra beklemeden hızlı bir şekilde yapabilmeleri ve kantincinin ürettiği sıcak servis ürünlerini (çay, tost vb.) daha sağlıklı koşullarda hazırlayabilmesine imkân sağlanması hedefi ile donanım ve yazılım unsurlarını içinde barındıran uygulama tabanlı bir proje geliştirmek amaçlanmıştır. Araştırmada nicel araştırma yöntem ve tekniklerinden deneysel araştırma yöntemi kullanılmıştır. Araştırmada, kantin sırası problemini çözecek hipotezler geliştirilmiş ve belirli hedefler doğrultusunda test ve denemeler yapılmıştır. Bu proje ile uygulamalı bir araştırma ve deneysel bir geliştirme yapılmış, sonunda arge ve inovasyon ürünü ortaya çıkarılmıştır. Projede öğrenciler kişisel hesapları ile kiosklar ve internet üzerinden sipariş verir ve sıra beklemez. Hazırlanan ürünler kantinci tarafından kilitli kutulara yerleştirilir. Öğrenci siparişinin bulunduğu kutuyu kimliğini okutarak açar ve şiparişini alır. Böylelikle siparişini alırken de sıra beklememiş olur. Siparişin teslim süresi planlanabilir. Örneğin 2.tenefüs, 4.tenefüs gibi. Planlı yapılan alışveriş, kantincinin ürünü geniş zaman aralığında hijyenik olarak hazırlamasına imkan tanır.

**Comment by ecedeger** [comment]
**GÖRME ENGELLİLER İÇİN NESNE, MEKAN TANIMLAYICI YAPAY ZEKA SİSTEMİ VE VERİ SETİNİN GELİŞTİRİLMESİ**

Yapay zeka çalışmaları ülkelerin gelecek planlamaları içerisinde önemli bir yer tutmaktadır. İnsana ait fiziksel ve bilişsel özellikleri taklit etme yeteneğine sahip olan yapay zeka sistemleri özellikle hızlanan işlemciler ve maliyetlerin düşmesiyle birlikte eğitim, sağlık ve finans gibi yaşamın çeşitli alanlarında kendini göstermeye başlamıştır. Görme yetisini sonradan kaybetmiş veya doğuştan görme engelli olan kişiler hayatlarında risk ve zorluklarla karşılaşırlar. Yapay zeka teknolojileri ile görme engellilerin yaşantılarını kolaylaştırmak adına uygulama ve modeller oluşturulmuştur. Oluşturulan projelerin kullanım koşulları ve bu bağlamda yapılan uygulama sayısı sınırlı sayıdadır. Bu çalışmada günlük yaşamda bu sağlık problemiyle karşılaşan bireyler üzerinde hem kullanım bakımından kolay, hemde kapsamlı olarak bir yapay zeka projesi geliştirilmesi amaçlanmıştır. Sistemin mobil kısmı Android Studio 3.0 ortamında Java programlama dili ile geliştirilmiştir. Sistemin geliştirici kısmı ise PyCharm 2018.2.4 ortamında Python programlama dili ile geliştirilmiştir. Geliştirilen proje'de toplanan veri seti aynı zamanda yapay zeka geliştiricilerinin kullanması adına açık kaynak olarak paylaşılmıştır. Çalışma sonucunda oluşturulan Türkiye'nin ilk Lise öğrencileri tarafından oluşturulan dış mekan veri setinde bulunan sınıflar doğrultusunda (Eczane,atm vb.) görme engelli bireye sesli bir şekilde hem mesafe olarak hem de sesli olarak geri bildirim veren aynı zamanda kendi kendine öğrenebilen bir yapay zeka uygulaması gerçekleştirilmiştir.

**Comment by PourToujoursMysterieux** [comment]
VİGENERE ŞİFRELEME YÖNTEMİNİN TEK VEYA ÇİFT BÖLEN SAYISINA GÖRE KRİPTO SIRALI İKİLİLERİ İLE ŞİFRELENMESİ
1586 yılında Fransız diplomat Blaise de Vigenere tarafından vigenere şifreleme yöntemi geliştirilmiştir. Bu şifreleme yöntemine göre önce alfabemizin harfleri sırasıyla yazılır ve her harfi numaralandırılır(A=0,B=1,C=2???.gibi )daha sonra kullanacağımız anahtar kelime belirlenerek açık metne eklenir ve şifreleme işlemi gerçekleştirilir. Vigenere şifreleme türü yöntemlerde, şifreleme ve şifre çözümü için aynı anahtar kullanılmaktadır yani metni şifreleme için gizli bir anahtar seçilir ve bu gizli anahtarla metin şifrelenir. Şifrelenmiş metin alıcı kişiye yollanır ve gizli anahtar da gizli bir şekilde alıcı kişiye bildirilir. Şifrelenmiş metni alan kişi metni gizli anahtarla çözüp düz metne çevirir ve metni okur. Vigenere şifreleme işleminde kullanılan anahtar boyutu eğer bilinirse şifre frekans analizine tabi tutularak deşifre edilebilir. Biz projemizde vigenere şifreleme yönteminin kırılabilmesini daha da zorlaştıracak Kripto sıralı İkili Sayıları adını verdiğimiz bir yöntemle şifreleme işlemi gerçekleştirdik bu yöntemle bir tam sayının tek bölenleri sayısı (1 hariç)veya bir tam sayının çift bölenleri sayısından yola çıkarak çok alternatifli bir şifreleme yöntemi gerçekleştirdik ve yaptığımız literatür çalışmasında böyle bir çalışmanın yapılmadığını gördük Kripto sıralı ikililerinde ikililerden hangisinin tek bölenleri sayısını(1 hariç) veya çift bölenleri sayısını alacağımızı şu şekilde anlayacağız: (a',b'] ifadesinde a sayısının pozitif çift bölen sayısını ; b sayısının pozitif tek bölen sayısını(1 hariç) alacağımızı [a',b') ifadesinde a sayısının pozitif tek bölen sayısını(1 hariç);b sayısının pozitif çift bölen sayısını alacağımızı (a',b') ifadesinde a ve b sayılarının pozitif çift bölen sayısını alacağımızı [a',b'] ifadesinde a ve b sayılarının pozitif tek bölen sayılarını alacağımızı göstermektedir. Not: (a,b]aralığındaki sayılar ifadesiyle karıştırılmaması amacıyla biz Kripto sıralı İkili Sayılarını (a',b'] şeklinde tanımladık.

**Comment by kaanpinar04** [comment]
PARA PROBLEMİ VE DİNAMİK PROGRAMLAMA İLE ASİMETRİK ŞİFRELEME
Paraların bozdurulması probleminin sayıların parçalanması ile sıkı bir ilişkisi vardır. Para problemi (coin problem) olarak bilinen bu problemlerde bir sayının verilen sayılarla ifadelendirilmesinde mümkün olmayan parçalanmaların sayısı araştırılmaktadır. Dinamik Programlama optimumluk ilkesine dayalı çözümleme yöntemlerine verilen addır. Çalışmamızda Para Probleminin Dinamik Programlama yöntemiyle çözümünden yola çıkarak belirlenen alfabedeki harf sayısı Dinamik Programlama ile çözülmüştür. Daha sonra alfabedeki harf sayısı olan p ile hesaplanan tabloya (mod p) uygulanarak alfabenin tüm sayısal karşılıkları birden çok frekanslarla farklı indislerle tabloda elde edilmiştir. Asimetrik şifreleme algoritmaları sayesinde alıcı ve verici taraflar kendilerine ait ortak gizli anahtar oluşturabilirler ve verilerini bu anahtarla şifreleyebilirler. Ortak gizli anahtarı oluşturmak için ilk ortaya atılan Diffie-Hellman algoritması çalışmamızda kapalı anahtarın oluşturulmasında kullanılmıştır. p 'nin ilkel kökleri kullanılarak Diffie-Hellman algoritması ile k anahtarı elde edilmiştir. Açık metin k anahtarına göre düzenlenerek ve Kripto Dinamik Şifreleme Tablosu ile Polybius Şifresi benzeri bir yöntemle kapalı metin elde edilir. Deşifre formülü ile metin açık metne çevrilir. Çalışmamızda hem simetrik hem de asimetrik şifreleme algoritmaları kullanılmıştır. Tablonun oluşturulması ve haberleşme için C programlama dili ile kodlar yazılmıştır (Ek 2 ve Ek 3). Program çalıştırılarak şifre ve deşifre işlemleri yapılmıştır. Çalışmamız asimetrik şifreleme algoritmasını kullanır fakat şifreli metin için alanyazın taramasında bulunmayan tamamen bizim geliştirdiğimiz bir tablo oluşturma yöntemiyle elde edilen tablo işlevsel olarak kullanılmak istendiği için Kripto Dinamik Şifreleme Tablosu ile şifreleme tercih edilmiştir. Bu yanıyla yöntemimiz orijinaldir. Yapılan çalışmada çözülmesi zor bir algoritma geliştirilmeye çalışılmış olup bilgi güvenliği alanına katkı sağlaması beklenmektedir.

**Comment by TahaGuzell** [comment]
CEP LABORATUVARI
Bu çalışmada, kimya laboratuvarına erişimi ekonomik imkansızlıklar, bilgi yetersizlikleri ve güvenlik önlemlerinin yeterli olmaması gibi nedenlerden dolayı laboratuvara erişimi kısıtlı olan öğrenciler için sanal bir kimya laboratuvarı oluşturmak amaçlanmıştır. Bu amaç doğrultusunda; YÖK Tez, Google Akademik ve DergiPark gibi kaynaklardan literatür taraması yapılmış, Laboratuvar ortamına gidilip gözlem yapılmış, kimya ve sanal gerçeklik alanında uzman kişilerle görüşmeler gerçekleştirilmiş, Hazırlık, 9.Sınıf ve 10.Sınıf öğrencilerine Kimya dersi ile ilgili anket çalışması yapılmıştır. Ardından MEB 9. sınıf kimya müfredatı kazanımlarının bulunduğu, sanal bir laboratuvar ortamı içerisinde toplamda dört bölümden oluşan eğitsel bir sanal gerçeklik uygulaması "Unity 3D" üzerinden geliştirilmiş ve uygulamaya "Cep Laboratuvarı" ismi verilmiştir. Cep Laboratuvarına ait 3B modeller "Blender 3.0" üzerinden tasarlanıp, ses kayıtlarının stüdyo ortamında kaydedilmesinin ardından tüm bunlar Unity platformuna aktarılmıştır. Çalışma kapsamında, geliştirilen uygulama ana hedef kitlesi olan 9. sınıf öğrencilerine uygulanmıştır ve uygulamaya olan yaklaşımları gözlemlenmiştir. Bu tarz içinde yeni nesil teknolojileri barındıran eğitim materyallerinin ilgi çekmesinin yanı sıra akademik başarıya destek sağladığı tespit edilmiştir.

**Comment by zbarramm** [comment]
TÜKENMEZ KALEM (CİDDİ ANLAMDA) 

Kalemler genelde bizlerin kullanıp bitince attığımız ürünler olurlar. Zaten bitmiş bir kalemle ne yapılabilir ki? Adı üstünde bitmiş, işlevini çoktan yitirmiştir bu kalem. İşe yaramaz bir plastik veya yapıldığı malzeme her neyse odur, başka bir şey değildir. Tabii sizler onu saça tutturmak, koleksiyona koymak gibi farklı alanlarda kullanmazsanız geçerlidir bu. En basitinden düşünecek olursak plastik bir kalem için önce ham madde ardından işçilik gerekir. Günümüzde artan makineler sayesinde bizler bir kalemi her ne kadar kısa sürede üretebiliyor olsak dahi kalemleri hiçten saymamalıyız. Demek istediğim kalem için harcanan bir zamanın ve emeğin olduğunu unutmamalıyız. Az önce de bahsettiğim gibi kalemler biterler en azından pek çoğunun ömrünün uzun olduğu söylenemez. Bu projeyse kalemlerin geri dönüşümüne, yeniden kullanılabilirliğine nasıl katkıda bulunabileceğimiz. Kalemler hızlı tükeniyor veya kayboluyorlar. Bir yılda belki   milyonlarca  kalem çöpe atılıyor. Bir iki kalem bizler için çok büyük bir kayıp olmasa dahi tüm dünyayı düşündüğümüzde çok fazla, haddi hesabı olmayan bir tüketimle karşılaşıyoruz. Bu proje sayesinde kalemler daha uzun süre kullanılabilecek. Kalemlerin daha uzun süre kullanılması demek ham maddeden, enerjiden, işçilikten ve paradan daha az kullanılması demektir. Bu kalemlerin içinde özel hazneleri bulunuyor. Bu hazneler sayesinde yeniden doldurulabilme özellikleri oluyor. Kalemler tekrar tekrar kullanılabiliyor. Bu tekrar kullanımıysa içinde bulunan sistem ve katı kalem mürekkebi sağlıyor. Katı kalem mürekkebi hem etrafı kirletmiyor hem de sıvısı döküldüğü zaman kalemdeki sistemin tuşuna basınca kalemin haznesinin içindeki aparatın herhangi bir güç gerektirmeden kendini kilitleyip karıştırmasıyla renk veriyor. Bu kalemlerin farklı uç ve renk seçenekleri de mevcut. Bu sayede çok az sayıda kalemle size bir ömür yetecek ve bittiğinde yeniden kullanılabilecek kalemler daha kullanışlı oluyorlar. İçindeki sistem sayesinde geliştirilebilen bu kalemler kayboldukları zaman anahtar kod olan sözcüğü iki kez söylediğinizde kendini belli ediyor.

**Comment by adasue** [comment]
Trajectoid
Tekrarlanan herhangi bir yol için bu yolu kendi başına takip edecek şekilde yuvarlanabilecek bir cisim tasarlamak mümkün müdür? Peki böyle bir şeyi yapmanın bize nasıl bir faydası olabilir? Fen ve teknolojide liderlik yapan bir üniversite olan UNIST’e bağlı bir araştırma ekibi günümüzde yeni bir çağ açabilecek çalışmalara temel atabilecek bir araştırma yapıyor. “Trajeoktidler”(İngilizce adıyla trajectoids) adı verilen bu nesneler tam olarak bunu yapabiliyor. Yalnızca basit yollar olmamakla beraber örneğin sadece bir doğru ya da bir eğri değil de daha karmaşık ve iç içe geçmiş yollarda da işe yarayacak şekilde kendi başına yuvarlanarak bu yolları takip edebilecek bu cisimleri yaratıyorlar. Gideceği yolun şekline ve bazı hesaplamalara göre şekilleri verilen bu cisimler şaşırtıcı bir şekilde sadece ilk başta ittirilmeleriyle kendi başlarına istenilen yolun şekli ve yörüngelerine göre ilerliyorlar. Bunun nerede kullanılabileceğine gelirsek de mayın tarlalarında veya belirlenen bölgelerde askeri amaçlı mayın kontrolü yapmak için tüm bölgeyi taramak amacıyla kullanılabilir. Boyutlarına ve taranan bölgenin alanının hesaplamalarına göre şekillendirilebilecek bu şekilli cisimler sayesinde şu anda mayın kontrolü yapmak için kullanılan çoğu tekniğe göre daha pratik, doğru ve güvenilir bir yöntemle mayınların yerleri tespit edilebilir.

**Comment by sonsuzus** [comment]
Bu konu neden 10 kişi ile sınırlı kaldı?

**Comment by KayraMertAydin** [comment]
**SESLİ PROSPEKTÜS**
Günümüzde internet kullanımı biçimi hızla gelişerek yeni teknolojilere evrilmektedir. Artık hemen hemen bütün işlemlerimizi akıllı telefon üzerinden kontrol edebildiğimiz bir teknoloji hayatımızı her alanda kuşatmaya başlamıştır. Bu çalışmanın amacı, ilaçların uzun prospektüslerini insanların okuma zahmetinden kurtarmaya yöneliktir. İlaç kullanım oranı yaşın ilerlemesi ile birlikte artmaktadır.Yaşın ilerlemesi ile birlikte insanlarda görme problemleri başlamaktadır. İlaçların prospektüslerinde çok küçük puntoyla bu yazılar yazıldığından okumakta sıkıntı çekildiği ortadır. Bu amaç doğrultusunda, android işletim sistemine sahip cihazlar üzerinden ilaçların Barkod bilgisini okutarak ilaçların önemli bilgileri olan kullanım şekli ve yan etkileri kısımlarının sesli olarak okutulması gerçekleştirilmiştir. Projede, barkod okuma kütüphaneleri ve Google metin okuma (Text To Speech - TTS) özelliği kullanılarak Android Studio yazılımı ile geliştirme yapılmıştır. Programda ilaç üzerindeki Barkod bilgisi okutularak veri tabanındaki eşleşme doğrultusunda ilgili ilaç fotoğrafıyla birlikte ekranda gözükerek ilaç bilgilerini dinle butonuna bastığımızda ilaç bilgilerini sesli olarak okumaya başlamaktadır. Bu projenin odak kitlesi yaşlılar olsa da geniş bir odak kitlesine sahip olmaktadır. Akıllı telefon kullanan her birey bu uygulamadan yararlanabilmektedir.
